csrankings-l.csv

```diff
@@ -257,8 +257,10 @@ Leliane Nunes de Barros,USP,http://www.ime.usp.br/~leliane,CqqZ8b0AAAAJ
 Len Adleman,University of Southern California,https://www.usc.edu/dept/molecular-science/fm-adleman.htm,NOSCHOLARPAGE
 Lena Mashayekhy,University of Delaware,https://www.eecis.udel.edu/lena,b9gs2WYAAAAJ
 Lena Peterson,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/lenap.aspx,NOSCHOLARPAGE
+Lénaïc Chizat,EPFL,https://people.epfl.ch/lenaic.chizat,jrJh9yIAAAAJ

```

Comment:
Not shown as being able to supervise CS students; please clarify how they meet the inclusion criteria.